nifi-commons/nifi-properties/src/main/java/org/apache/nifi/util/NiFiProperties.java

```diff
@@ -354,6 +356,7 @@
     public static final String DEFAULT_SECURITY_USER_SAML_HTTP_CLIENT_TRUSTSTORE_STRATEGY = "JDK";
     public static final String DEFAULT_SECURITY_USER_SAML_HTTP_CLIENT_CONNECT_TIMEOUT = "30 secs";
     public static final String DEFAULT_SECURITY_USER_SAML_HTTP_CLIENT_READ_TIMEOUT = "30 secs";
+    private static final String DEFAULT_SECURITY_USER_JWS_KEY_ROTATION_PERIOD = "PT1H";
```

Comment:
Do you think this should be changed to used the time period format of other properties? It is a new property that stands alone, and it could provide a path forward for other new properties.

Reply:
This format is preferable.  Just suggesting this conversion for all properties in the context of a potential 2.0 release.